### ISSUE TITLE
Feature Request: Click to open job

### ISSUE THREAD

**Issue by bennypowers** [blakawk/gitlab-integration#34]
It'd be great to be able to click on a job bubble and be taken to that job in the default browser

**Comment by jasondamour** [comment]
What work needs to be done? I'm willing to sink a couple hours into it, if you could point me in the starting direction @blakawk

**Comment by bennypowers** [comment]
@jdamour98  probably would want to change this to a link to the relevant url

https://github.com/blakawk/gitlab-integration/blob/master/lib/status-bar-view.coffee#L103

**Comment by blakawk** [comment]
> What work needs to be done? I'm willing to sink a couple hours into it, if you could point me in the starting direction @blakawk

Indeed, as @bennypowers stated, @jdamour98 you should update the stage tooltip in order to display the list of the jobs with there status (the best would be to display also their respective status like shown below):
![image](https://user-images.githubusercontent.com/1149069/59335837-a4bd6180-8cfd-11e9-9a22-cf0b607f836d.png)

Then, use the `job.web_url` attribute to construct the `<a>` element for each job in the stage tooltip.

**Comment by blakawk** [comment]
@jdamour98 Refer to https://atom.io/docs/api/v1.38.0/TooltipManager for documentation about how to make a tooltip an HTML element containing the list of jobs.
